lib/entangler/executor/slave.rb

```diff
@@ -12,7 +12,7 @@ def initialize(base_dir, opts = {})
 
         @remote_reader = $stdin
         @remote_writer = $stdout
-        $stderr.reopen(File.join(Entangler::Logger.log_file_path(base_dir, 'entangler.err')), 'w')
+        $stderr.reopen(File.join(Entangler::Logger.log_file_path(base_dir, 'entangler.err')), 'a')
```

Comment:
This might create issues like https://github.com/daveallie/entangler/pull/18 but we can deal with that later.